ui/src/index.css

```diff
@@ -15,3 +15,49 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.up-arrow {
```

Comment:
Do we need this CSS. Can this be handle by tailwind?

Reply:
Yes would like go with this CSS now, Checked tailwind wasn't really straight forward. It would be great if you can create another issue for this

Reply:
Suggestion: Optimized CSS as of now. You have a repeated style for the cursor. You can optimise it. If you need any help just let me know

Reply:
@Neha made some corrections, Let me know if i should put more thought into it. Before i ask you for help

Reply:
Please change CSS like this:

```
.up-arrow , .down-arrow{
  border: solid 6px transparent;
  background: transparent;
  border-top-width: 0;
  cursor: pointer;
}

.clicked-d{
  border-top: solid 7px #ad9f9f;
}

.clicked-u{
  border-bottom: solid 7px #ad9f9f;
}

.down-arrow{
  margin-top:1px;
}
```